core/internal/featurechecker/features.go

```diff
@@ -0,0 +1,94 @@
+package featurechecker
+
+import (
+	"context"
+	"sync"
+
+	"github.com/Khan/genqlient/graphql"
+	"github.com/wandb/wandb/core/internal/gql"
+	"github.com/wandb/wandb/core/internal/observability"
+	spb "github.com/wandb/wandb/core/pkg/service_go_proto"
+)
+
+// ServerFeaturesCache is responsible for providing the capabilities of a server
+// Features and capabilities are retrieved from the server via a GraphQL query
+type ServerFeaturesCache struct {
+	Features map[spb.ServerFeature]Feature
+
+	ctx           context.Context
+	graphqlClient graphql.Client
+	logger        *observability.CoreLogger
+	once          sync.Once
+}
+
+type Feature struct {
+	DefaultValue string
```

Comment:
should it be just Value?

Reply:
Hmm I have decided to remove this for now, since we do not have a way to get default value from the backend server I think this will cause too much confusion anyways.